tools/cvm-image-rewriter/plugins/06-install-tdx-guest-kernel/README.md

```diff
@@ -0,0 +1,22 @@
+# Install TDX guest kernel
+
+This plugin is used to install TDX guest kernel from a given local repository.
+
+# Prerequisite
+
+Prepare the local repository and confirm that there are Debian packages related to the TDX kernel in the `/jammy/amd64/` directory of this repository. You can choose to place this local repository in the `pre-stage/artifacts/` directory.
+```
+mkdir -p ./pre-stage/artifacts
+mv <your guest repo> ./pre-stage/artifacts/
+```
+
+Set `${CVM_TDX_GUEST_REPO}` to the repository absolute path, or this plugin will be skipped. 
+```
+export CVM_TDX_GUEST_REPO=$(pwd)/pre-stage/artifacts/<your guest repo>
+
+# Or
+export CVM_TDX_GUEST_REPO=<your local guest repo>
+```
+
+
+_NOTE: IF the original image is smaller than 1.5G, please also set the environment variable GUEST\_SIZE larger ._
```

Comment:
It might be worth mentioning this will cause plugin 01 to run by setting this variable.